products/bridge/bridge-validators/Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: all
+all: image/build-and-push
+
+.ONESHELL:
+SHELL := /bin/bash
+.SHELLFLAGS = -ec
+
+IMAGE_TAG ?= localhost:5001/zilliqa-bridge-validator:latest
```

Comment:
the `zilliqa-bridge-validator` placeholder, is this what i specified in the github workflow?